gazebo/rendering/ScenePrivate.hh

```diff
@@ -51,6 +51,9 @@ namespace gazebo
 {
   namespace rendering
   {
+#if OGRE_VERSION_MAJOR == 1 && OGRE_VERSION_MINOR <= 9
+    class CustomSceneManagerFactory;
+#endif
```

Comment:
I think this doesn't technically need version macros

Reply:
fixed